utils/utils.ts

```diff
@@ -331,11 +331,12 @@ export async function getThirdPartyRewards(
   rewardToken: BN,
 ) {
   const api = getApi();
-  return await api.rpc.pos.calculate_3rdparty_rewards_amount(
+  const calculation = await api.rpc.pos.calculate_3rdparty_rewards_amount(
     userAddress,
     liquidityAssetId.toString(),
     rewardToken.toString(),
   );
+  return new BN(calculation.toString());
```

Comment:
use the function stringToBN instead. when large numbers, toString returns a 0x, and new BN will fail.

Reply:
Done